doc/src/modules/physics/units/examples.rst

```diff
@@ -101,7 +101,7 @@ variables (taken from Wikipedia). The result should be 224.701 days.
     Eq(T**2/venus_a**3, 4*pi**2/(gravitational_constant*solar_mass))
     >>> q = solve(eq, T)[1]
     >>> q
-    2*pi*venus_a**(3/2)/(sqrt(gravitational_constant)*sqrt(solar_mass))
+    2.0*pi*venus_a**(3/2)/(sqrt(gravitational_constant)*sqrt(solar_mass))
```

Comment:
this should not happen

Reply:
@Upabjojr It gave an error there, failed example